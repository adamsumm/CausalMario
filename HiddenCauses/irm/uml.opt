; run-specific parameters for the Infinite Relational Model
;
--loops=20
--nchains=1
--mcmcflag=0
--outroot=output/uml
--configfile=data/uml.config
--graphname=data/uml.graph
--initfile=
--hypupdates=5
--betamagupdate=1
--betapropupdate=0
--alphaupdate=1

