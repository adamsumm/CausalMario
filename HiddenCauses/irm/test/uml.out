   0: -38623.995  -3.892  -4.905    1    1 -38632.792   1.000   0.200   1.000   1.000 
   1: -19529.449 -116.931 -291.986   17   11 -19938.367   1.000   0.200   1.000   1.000 
   2: -14742.799 -116.063 -315.781   23   12 -15174.643   1.000   0.200   1.000   1.000 
   3: -14149.504 -111.697 -328.085   22   13 -14589.285   1.000   0.200   1.000   1.000 
